src/cache/LruCacheObject.js

```diff
@@ -43,4 +38,7 @@ export default class LruCacheObject {
       this._cacheOrdering.splice(index, 1);
     }
   }
+  isValidCacheKey(cacheKey) {
+    return typeof cacheKey === 'string' || typeof cacheKey === 'number';
```

Comment:
Consider breaking this out into a `isValidObjectKey` function.

Reply:
✅